pkg/api/v20210131preview/openshiftcluster_validatestatic_test.go

```diff
@@ -462,6 +463,63 @@ func TestOpenShiftClusterStaticValidateNetworkProfile(t *testing.T) {
 	runTests(t, testModeUpdate, tests)
 }
 
+func TestOpenShiftClusterStaticValidateNetworkProfileType(t *testing.T) {
```

Comment:
I think lets put it into `TestOpenShiftClusterStaticValidateNetworkProfile`.

Reply:
Tests moved.